bundles/org.openhab.binding.modbus/src/main/java/org/openhab/binding/modbus/internal/handler/ModbusDataThingHandler.java

```diff
@@ -513,8 +517,9 @@ private void validateAndParseReadParameters(ModbusDataConfiguration config) thro
         if (childOfEndpoint && readRequest == null) {
             if (!readStartMissing || !readValueTypeMissing) {
                 String errmsg = String.format(
-                        "Thing %s readStart=%s, and readValueType=%s were specified even though the data thing is child of endpoint (that is, write-only)!",
-                        getThing().getUID(), config.getReadStart(), config.getReadValueType());
+                        "Thing %s was configured for reading (readStart and/or readValueType specified) but the "
+                                + "parent bridge is not a polling one. Consider using a bridge of type 'Regular Poll'.",
```

Comment:
Please avoid concatenating strings for no reason and rather define only one string with the full text.

Reply:
@lolodomo You mean to have a single line string (thus spanning >150 columns) instead of splitting it into two lines? The reason for the concatenation would have been to avoid long code lines.